src/script/auth/component/HasToUseDesktop.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+
+import {useIntl} from 'react-intl';
+
+import {Runtime} from '@wireapp/commons';
+import {Container, H2, H3} from '@wireapp/react-ui-kit';
+
+import {WirelessContainer} from './WirelessContainer';
+
+import {Config} from '../../Config';
+import {unsupportedJoinStrings, unsupportedStrings} from '../../strings';
+
+const hasToUseDesktopApplication = () =>
+  !Runtime.isDesktopApp() && Config.getConfig().FEATURE.ENABLE_ENFORCE_DESKTOP_APPLICATION_ONLY;
+
+interface UnsupportedBrowserProps extends React.HTMLProps<HTMLDivElement> {
+  isTemporaryGuest?: boolean;
+}
+
+export const HasToUseDesktop = ({children, isTemporaryGuest}: UnsupportedBrowserProps) => {
```

Comment:
I dont like the naming of this component. Should rather be something like <SupportedBrowserWrapper> or anything like thas. 

has*, is* naming scheme indicates that the variable is a boolean.

Reply:
In the end, the component is not needed at all, as the same check is already happening in `src/script/main/index.tsx`